.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - 2.7
+before_install:
+  - export DISPLAY=:99
+  - sh -e /etc/init.d/xvfb start
+install:
+  - pip install -r test-requirements.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/base.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements/test.txt
+  - pip install -r $VIRTUAL_ENV/src/xblock/requirements.txt
```

Comment:
@smarnach @bradenmacdonald I don't have a specific proposal.   I wanted to know why you needed to install xblock-sdk's test requirements.  I would have thought they were only needed for testing the sdk, which you are not doing here.  I understand about the difficulty of managing packages, and I'm sure part of this is us being sloppy in the xblock-sdk repo.  I wanted to understand the problem so that we could decide on a way to fix it, wherever the best place is.